.idea/kotlinc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't know if it should be added to .gitignore, but at least remove it from this PR